src/mainwindow.cpp

```diff
@@ -400,6 +400,12 @@ MainWindow::MainWindow(const QString &fileName, QWidget *parent) :
     connect(ui->actionAboutQt, &QAction::triggered, qApp, &QApplication::aboutQt);
 
     setAcceptDrops(true);
+
+    //Set multiple shortcuts for clear action to add support for backspace
+    QList<QKeySequence> shortcuts;
+    shortcuts << QKeySequence("Ctrl+Del") << QKeySequence("Ctrl+Backspace");
```

Comment:
In UI LogLite would still show Cmd+Del as the shortcut key on the mac (I think it just picks up the first shortcut). There will be no indication that Cmd+Backspace is also a thing. Maybe make an ```#if mac``` - then Ctrl+backspace?

Reply:
Was trying to avoid if mac but looks like it is looking unavoidable, will change it